pkg/logger/mock.go

```diff
@@ -2,13 +2,38 @@ package logger
 
 import "time"
 
-// MockLogger is a logger meant to be used in testing. It records all logs sent
-// to it and provides useful fields for you to verify that your application
-// logs as expected.
-type MockLogger struct {
+// Mock is a logger and a sink meant to be used in testing. It records all
+// logs sent to it and provides useful fields for you to verify that your
+// application logs as expected.
+type Mock struct {
 	// Logs is an array of logs recorded. Each logged event is stored as a
 	// separate item in this array..
 	Logs []MockLog
+	// LogMessages is an array of log messages recorded. This complements the
+	// array of MockLog for easier assertion that expected messages has been
+	// logged. Empty messages are also stored in this array as empty strings.
+	LogMessages []string
+}
+
+// NewMock creates a new Logger interface compatible type that holds additional
+// fields of the logs that has been submitted.
```

Comment:
typo
```suggestion
// fields of the logs that have been submitted.
```

Reply:
more of a grammatical error than a typo :)